notebooks/explainable-ai-1-basic/explainable-ai-1-basic.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    output_layer = compiled_model.output(0)
</pre><p>it is old syntax</p><p>there is no need to obtain output_layer, enough just use compiled_model([input_image])[0] in next cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Fixed!